tests/integration/targets/sefcontext/tasks/sefcontext.yml

```diff
@@ -100,3 +105,129 @@
     that:
     - sixth is not changed
     - sixth.setype == 'unlabeled_t'
+
+- name: Set SELinux file context path substitution of foo
+  sefcontext:
+    path: /tmp/foo
+    equal: /home
+    state: present
+    reload: no
+  register: subst_first
+
+- assert:
+    that:
+    - subst_first is changed
+    - subst_first.equal == '/home'
+
+- name: Set SELinux file context path substitution of foo (again)
+  sefcontext:
+    path: /tmp/foo
+    equal: /home
+    state: present
+    reload: no
+  register: subst_second
+
+- assert:
+    that:
+    - subst_second is not changed
+    - subst_second.equal == '/home'
+
+- name: Change SELinux file context path substitution of foo
+  sefcontext:
+    path: /tmp/foo
+    equal: /boot
+    state: present
+    reload: no
+  register: subst_third
+
+- assert:
+    that:
+    - subst_third is changed
+    - subst_third.equal == '/boot'
```

Comment:
You mean to actually read the label from the disk? I guess that is possible. Should I add the same changes to the test that were not touched by this PR?

Reply:
Ah, just thought about this more. So we're not writing any test files to disk so that cannot be checked - and policy reloading is set to `false` so I doubt those changes are actually even committed.

We are just modifying the SELinux context mappings, not testing to write files to disk.

Currently I don't think it is possible to read back context mappings from the policy. That would probably need a new feature - like using this module with a new option like `state=read`.

Reply:
To actually test writing files there are a few different ways that could work:
1. Create a file/directory with `file` module and set `setype=_default`.
2. Create a file/directory at the beginning of the test and keep running `restorecon` to it after each test.

Both would require to change `reload=true`.

Having said that - I am not sure that is necessary. If a context mapping is accepted by `seobject` then that should be it, the rest is out of our hands anyways.

Reply:
okie, fair enough

Reply:
thanks for the explanation :-)